io/eolearn/io/sentinelhub_service.py

```diff
@@ -34,7 +36,7 @@ class SentinelHubOGCInput(EOTask):
     :type size_y: int or str, depends on the service_type
     :param maxcc: maximum accepted cloud coverage of an image. Float between 0.0 and 1.0. Default is ``1.0``.
     :type maxcc: float
-    :param image_format: format of the returned image by the Sentinel Hub's WMS getMap service. Default is 32-bit TIFF.
+    :param image_format: format of the returned image by the Sentinel Hub's WMS getMap service. Default is 16-bit TIFF.
```

Comment:
If you say that default is 16-bit that is not correct and confusing to the user because it is not the same as specifying `image_format=MimeType.TIFF_d16`.

Better to write more precise explanation:
*"Default is `None` which means values will be downloaded in 16-bit tiff but then cast into reflectance float values. The result is the same as by setting `MimeType.TIFF_d32f`, but with less data to download."*

Reply:
Right you are, will fix.